uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerHost.java

```diff
@@ -357,8 +357,17 @@ public void overwriteFlights(@Nullable final String flightsJson) {
         UiAutomatorUtils.obtainChildInScrollable("Overwrite flights (When BrokerHost is the active broker)");
         // input flights string in flights input box
         UiAutomatorUtils.handleInput("com.microsoft.identity.testuserapp:id/editTextFlights", flightsJson);
+        // Set to use LocalStorage Flights
```

Comment:
What happens if we don't set this?

Reply:
What is the default flights provider in BrokerHost? Is there any default?

Reply:
Some recent UI updates in brokerhost made it so that the update flight and overwrite flight buttons were not enabled unless this setting was used. looks like there are no default flights set